### PR TITLE
Satis documentation updates

### PR DESCRIPTION
**Pull request by JamesRezo**
This PR adds informations on missing or new properties of the satis configuration file.

### CODE REVIEW

File: doc/articles/handling-private-packages-with-satis.md [hunk 1]
```diff
@@ -220,10 +220,14 @@ following to your `satis.json`:
...
 
- * `directory`: the location of the dist files (inside the `output-dir`)
+ * `directory`: required, the location of the dist files (inside the `output-dir`)
  * `format`: optional, `zip` (default) or `tar`
  * `prefix-url`: optional, location of the downloads, homepage (from `satis.json`) followed by `directory` by default
  * `skip-dev`: optional, `false` by default, when enabled (`true`) satis will not create downloads for branches
+* `absolute-directory`: optional, a _local_ directory where the dist files are dumped instead of `outpur-dir`/`directory`
+* `whitelist`: optional, if set as a list of package names, satis will only dump the dist files of these packages
+* `blacklist`: optional, if set as a list of package names, satis will not dump the dist files of these packages
+* `checksum`: optional, `true` by default, when disabled (`false`) satis will not provide the sha1 checksum for the dist files
```
Comment (stof) [inline doc/articles/handling-private-packages-with-satis.md]: wrong indentation. It should have the same indentation than existing items

File: doc/articles/handling-private-packages-with-satis.md [hunk 2]
```diff
@@ -220,10 +220,14 @@ following to your `satis.json`:
 
 #### Options explained
 
- * `directory`: the location of the dist files (inside the `output-dir`)
+ * `directory`: required, the location of the dist files (inside the `output-dir`)
  * `format`: optional, `zip` (default) or `tar`
  * `prefix-url`: optional, location of the downloads, homepage (from `satis.json`) followed by `directory` by default
  * `skip-dev`: optional, `false` by default, when enabled (`true`) satis will not create downloads for branches
+ * `absolute-directory`: optional, a _local_ directory where the dist files are dumped instead of `outpur-dir`/`directory`
```
Comment (alcohol) [inline doc/articles/handling-private-packages-with-satis.md]: Typo in `outpur`

File: doc/articles/handling-private-packages-with-satis.md [hunk 3]
```diff
@@ -220,10 +220,14 @@ following to your `satis.json`:
...
+ * `whitelist`: optional, if set as a list of package names, satis will only dump the dist files of these packages
+ * `blacklist`: optional, if set as a list of package names, satis will not dump the dist files of these packages
+ * `checksum`: optional, `true` by default, when disabled (`false`) satis will not provide the sha1 checksum for the dist files
 
 Once enabled, all downloads (include those from GitHub and BitBucket) will be replaced with a _local_ version.
```
Comment (alcohol) [inline doc/articles/handling-private-packages-with-satis.md]: What does this line relate to now?

Comment (JamesRezo) [inline doc/articles/handling-private-packages-with-satis.md]: Do you mean that "downloads" is an ambiguous term ? s/downloads/dist files/ ?

Comment (alcohol) [inline doc/articles/handling-private-packages-with-satis.md]: No I mean, what does "once enabled" refer to? What enabled?

Comment (JamesRezo) [inline doc/articles/handling-private-packages-with-satis.md]: I guess "Once the `archive` properties set" ...

Comment (alcohol) [inline doc/articles/handling-private-packages-with-satis.md]: Ah, right.

**Comment by alcohol** [comment]
Thanks

**Pull request merged**